### PR TITLE
Navbar with styles

### PR DESCRIPTION
**Pull request by nikhil9302**
# Description

Added a navbar component, with the required styles from the inspiration website. This requires styled-components module
```
npm install styled-components
```

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Browsers Tested on?

- [x] Chrome
- [x] Firefox
- [x] Edge

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have checked the responsiveness of the component I built
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: components/Nav/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  .logo {
+    padding: 15px 0;
+    cursor: pointer;
+  }
+`
+
+const Navbar = () => {
+  return (
+    <Nav>
+      <div className="logo">
```
Comment (charles-vj) [inline components/Nav/Navbar.jsx:29]: - Give a bit more spacing to the top and left of the logo

File: components/Nav/RightNav.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      padding-top: 2em;
+      color: #fff;      
+    }
+  }
+`;
+
+const RightNav = ({ open }) => {
+  return (
+    <div className='overlay overlay-contentpush open'>
+    <Ul open={open}>
```
Comment (charles-vj) [inline components/Nav/RightNav.jsx:97]: - A bit of space needed between text and animation

File: components/Nav/RightNav.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        
+        .hover-underline-animation:hover:after {
+          transform: scaleX(1);
+          transform-origin: bottom left;
+        }               
+        `
+        
+      );
+    }else{
+      return ``;
```
Comment (charles-vj) [inline components/Nav/RightNav.jsx:64]: - Add the underline animation in the burger menu links as well

File: components/Nav/Burger.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+/*This code is divided into 3 parts, the Styled burgerbar formed when the screen become mobile size, when clicked its Open state becomes tru and is paased on RightBar to change the navigation view */
+const StyledBurger = styled.div`
+width: 2rem;
+height: 2rem;
+position: fixed;
+top: 15px;
+right: 20px;
+z-index: 500;
+display: none;
+  @media (max-width: 768px) {
```
Comment (charles-vj) [inline components/Nav/Burger.jsx]: - There's a few responsiveness issues

File: components/Nav/Burger.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    display: flex;
+    justify-content: space-around;
+    flex-flow: column nowrap;
+  }
+  div {
+    width: 2rem;
+    height: 0.25rem;
+    background-color: ${({ open }) => open ? 'red' : 'white'};
```
Comment (charles-vj) [inline components/Nav/Burger.jsx]: - Let the colour of the cross icon after opening RightNav remain white

File: components/Nav/RightNav.jsx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  }    
+  ${ function ({open}){
+    if (open==false) {
+      return (
+        `.hover-underline-animation {
+          display: block;
+          position: relative;    
+        }
+        
+        .hover-underline-animation:after {
```
Comment (charles-vj) [inline components/Nav/RightNav.jsx:29]: - Make sure the animation is consistent with the text

**Review by charles-vj — CHANGES_REQUESTED** [review]
Just a few responsiveness and animation changes required

File: components/Nav/Navbar.jsx (hunk 1 above)
Comment (nikhil9302) [reply components/Nav/Navbar.jsx:29]: ok

File: components/Nav/RightNav.jsx (hunk 2 above)
Comment (nikhil9302) [reply components/Nav/RightNav.jsx:97]: sure

File: components/Nav/RightNav.jsx (hunk 3 above)
Comment (nikhil9302) [reply components/Nav/RightNav.jsx:44]: but the inspiration site(https://experiencewelcome.com/) didnt have any animation for burger menu. Will do if you want to.

File: components/Nav/RightNav.jsx (hunk 6 above)
Comment (nikhil9302) [reply components/Nav/RightNav.jsx:29]: it should have worked with display: block; in line 19 , but it didnt any alternative suggestion?

File: components/Nav/Burger.jsx (hunk 5 above)
Comment (nikhil9302) [reply components/Nav/Burger.jsx]: ok 👍

File: components/Nav/Burger.jsx (hunk 4 above)
Comment (nikhil9302) [reply components/Nav/Burger.jsx]: ohk, increasing the max-width should fix it?

Comment (charles-vj) [reply components/Nav/Burger.jsx]: Yea, it should

File: components/Nav/RightNav.jsx (hunk 6 above)
Comment (charles-vj) [reply components/Nav/RightNav.jsx:29]: Try centering the text in the middle and see if it looks good

**Comment by nikhil9302** [comment]
did the required changes hopefully, let me know if there are any other issues

**Review by charles-vj — APPROVED** [review]

**Review by charles-vj — APPROVED** [review]

**Review by joshi008 — APPROVED** [review]

**Pull request merged**